docs/sample-scripts/excel-scripts.yaml

```diff
@@ -1328,6 +1365,37 @@
       // Switch the view to the new worksheet.
       newSheet.activate();
     }
+'ExcelScript.RangeFill:interface':
+  - |-
+    /**
+     * This script sets the fill color of the used range to green.
+     */
+    function main(workbook: ExcelScript.Workbook)
+    {
+      // Get the used range of the current worksheet
```

Comment:
@AlexJerabek Those missing periods are repeated in excel-scripts.yaml and snippets.yaml. Sorry I didn't mention this earlier!

Reply:
I'll fix that in an upcoming PR before this gets merged to live.